rocAL_pybind/examples/rocAL_api_video_labels_pipeline.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2018 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import torch
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import numpy as np
+from parse_config import parse_args
+import os
+
+class ROCALVideoIterator(object):
+    """
+    ROCALVideoIterator for pyTorch.
+
+    Parameters
+    ----------
+    pipelines : list of amd.rocal.pipeline.Pipeline
+                List of pipelines to use
+    size : int
+           Epoch size.
+    """
+
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, display=False ,sequence_length=3):
+
+        try:
+            assert pipelines is not None, "Number of provided pipelines has to be at least 1"
+        except Exception as ex:
+            print(ex)
+
+        self.loader = pipelines
+        self.tensor_format = tensor_layout
+        self.multiplier = multiplier if multiplier else [1.0, 1.0, 1.0]
+        self.offset = offset if offset else [0.0, 0.0, 0.0]
+        self.reverse_channels = reverse_channels
+        self.tensor_dtype = tensor_dtype
+        self.batch_size = self.loader._batch_size
+        self.w = self.loader.getOutputWidth()
+        self.h = self.loader.getOutputHeight()
+        self.n = self.loader.getOutputImageCount()
+        self.rim = self.loader.getRemainingImages()
```

Comment:
What is remaining images means to video? Please change the name to describe correctly (Is it same as dataset_size?)

Reply:
Renamed to `self.remaining_sequences = self.loader.getRemainingImages()`